### DISCUSSION TITLE
Hide Create Button Operation

### DISCUSSION THREAD

**Discussion by hellodit** [avo-hq/avo#2658] [Help] [answered]
I need help, after searching in the documentation I don't find any guide to do this, for the context I'm using AVO latest version.

It's possible to hide this button and create operation on specific resource management? 

![image](https://github.com/avo-hq/avo/assets/32544683/6706ffc8-9744-4734-a9c2-50ecde47ddcb)

**Comment by Paul-Bob** [comment, marked as answer]
Hello @hellodit This can be achieved by two main ways:

## Authorization ( Pro license required )

Using authorization you can create a policy for the `Setting` resource then make [`create?`](https://docs.avohq.io/3.0/authorization.html#create) and [`new?`](https://docs.avohq.io/3.0/authorization.html#new) return `false`.

## Custom controls ( Advanced license required )

Using [`self.index_controls`](https://docs.avohq.io/3.0/customizable-controls.html#index_page) you can costumize whatever control appear on the index page. To hide the create button you should declare only `actions_list` and `attach_button` (for associations).
